keylime/cloud_verifier_common.py

```diff
@@ -76,6 +92,7 @@ def __exit__(self, *args):
             print('elapsed time: %f ms' % self.msecs)
 
 
+@convert_exceptions
```

Comment:
What is the reason to wrap the whole function with convert_exceptions vs using a try/except block around the ssl functions which are throwing the exception? IMO it's easier to see what's going on when the try/except is closer to the code that will throw the exception.

Reply:
Since this method is ssl operation intensive, the intent to have an exception wrapper is to have a common exception handling logic that can be extended, currently we only handle this special exception though. It works for me if we just try-catch load_cert_chain at the moment.